pipeline/errors/error.go

```diff
@@ -82,3 +82,38 @@ func HasBlockingErrors(err error) bool {
 
 	return false
 }
+
+var (
+	// ErrSkip is used as a return value when container execution should be
+	// skipped at runtime. It is not returned as an error by any function.
+	ErrSkip = errors.New("Skipped")
```

Comment:
as per go recommendations error text should start with lowercase letter
```suggestion
	ErrSkip = errors.New("skipped")
```